snake.asm

```diff
@@ -46,12 +46,9 @@ je .food                ; If it's food, place new food and skip tail removal
 es lodsw                ; Load the previous position (tail) from the stack into ax
 ; The instruction above retrieves the last position of the snake's tail from the stack,
 ; using the fact that si was set to point to the stack (mov si, sp)

```

Comment:
Line 47 is redundant the comment can just be "Load the previous position (tail) from the stack into ax using the fact that si was set to point to the stack (mov si, sp)"